index.html

```diff
@@ -0,0 +1,184 @@
+        <img
+          class="blck-nav-img"
+          src="images/facebook.png"
+          alt="facebook-icon"
+        />
+        <img class="blck-nav-img" src="images/twitter.png" alt="twitter-icon" />
+        <p class="blck-nav-text">English</p>
+        <p class="blck-nav-text">My Page</p>
+        <p class="blck-nav-text">Logout</p>
+      </div>
+    </nav>
+    <nav class="navbar">
+      <img class="Logo" src="images/GDC.png" alt="Logo" />
+      <div class="navbar-links">
+        <ul>
+          <li><a href="about.html">ِِAbout</a></li>
+          <li><a href="#">ِِProgram</a></li>
+          <li><a href="#">ِِJoin</a></li>
+          <li><a href="#">ِِSponser</a></li>
+          <li><a href="#">ِِNews</a></li>
+          <li><a class="active-border" href="#">ِِGDC Campaign</a></li>
+        </ul>
+      </div>
+    </nav>
+    <header>
+      <nav>
+        <div class="hambuger">
+          <img
+            class="menu-bar"
+            id="hamburger"
+            src="images/menu.png"
+            alt="menu"
+          />
+        </div>
+      </nav>
+      <div class="toggle-menu">
+        <img src="images/Icon.png" alt="close toggle" id="togclose" />
+        <div class="hambug-items">
+          <ul class="menu">
+            <li class="menu-list">
+              <a href="About.html" class="menu-link">ِِAbout</a>
+            </li>
+            <li class="menu-list">
+              <a href="#" class="menu-link">ِِProgram</a>
+            </li>
+            <li class="menu-list">
+              <a href="#" class="menu-link">ِِJoin</a>
+            </li>
+            <li class="menu-list">
+              <a href="#" class="menu-link">ِِSponser</a>
+            </li>
+            <li class="menu-list">
+              <a href="#" class="menu-link">ِِNews</a>
+            </li>
+            <li class="menu-list">
+              <a href="#" class="menu-link active-border">ِِGDC Campaign</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </header>
+    <section class="first-section">
+      <div class="transparent">
+        <div class="first-content">
+          <h4 class="text-hello">"Hello! Sharing World"</h4>
+          <h1 class="text">Game Developers Conference 2022</h1>
+        </div>
+        <div class="describe">
+          <p class="text2">Connecting the Global Game Development Community</p>
+        </div>
+        <div class="describe2">
+          <h2 class="text3">2022.3.21(MON) ~ 25 (FRI) San Francisco, CA</h2>
+        </div>
+      </div>
+    </section>
+    <section class="second-section">
+      <div class="second-content">
+        <h2 class="text4">Main Program</h2>
+        <div class="line"></div>
+      </div>
+      <div class="flexbox">
+        <div class="item">
+          <img
+            src="images/lecturer.png"
+            class="item-img margin0"
+            alt="lecturer"
+          />
+          <h2 class="item-title margin0">Lecture</h2>
```

Comment:
- Headings are meant to have a bigger font size with respect to the text around them. Kindly increase the font size of the Main Program Headings(Lecture, Exhibition, Forum, Workshop, and Ignite). This will improve user experience as it would allow users to see the main things/activities that would be done 👍